lib/logstash/filters/aggregate.rb

```diff
@@ -202,13 +215,20 @@ def filter(event)
 			map = aggregate_maps_element.map
 
 			# execute the code to read/update map and event
-			@codeblock.call(event, map)
+			begin
+				@codeblock.call(event, map)
+				noError = true
+			rescue Exception => exception
```

Comment:
Hi,
  is actually more idiomatic to rescue StandardError descendant exceptions here. You can use something like:

```
  begin 
     ....
  rescue => e
  end
```

If you want more details I found this article on the internet: https://robots.thoughtbot.com/rescue-standarderror-not-exception